assets/src/edit-story/elements/square/display.js

```diff
@@ -8,40 +8,30 @@ import styled from 'styled-components';
  * Internal dependencies
  */
 import {
-	ElementWithPosition,
-	ElementWithSize,
+	ElementFillContent,
 	ElementWithBackgroundColor,
-	ElementWithRotation,
 } from '../shared';
 
 const Element = styled.div`
-	${ ElementWithPosition }
-	${ ElementWithSize }
-	${ ElementWithRotation }
+	${ ElementFillContent }
 	${ ElementWithBackgroundColor }
 `;
 
-function SquareDisplay( { backgroundColor, width, height, x, y, rotationAngle } ) {
+function SquareDisplay( { backgroundColor, width, height } ) {
 	const props = {
 		backgroundColor,
 		width,
```

Comment:
Should width and height be removed here as well? (It's mainly a placeholder element anyway, so probably not that important)

Reply:
Good point. Removed.